.github/workflows/php.yml

```diff
@@ -44,12 +44,21 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Install Dovecot
+      run: |
+        sudo apt-get update -q
+        sudo apt-get install -yq dovecot-core
+        doveconf
+        cat /etc/passwd
```

Comment:
can you drop the cat / sudo lines please?

Reply:
It will clean up eventually, but sudo installation of dovecot is necessary.